common/journal_entries/imperia_qing_journal_entries.txt

```diff
@@ -224,7 +224,6 @@ je_investigate_military_reform = {
 			var:eight_banners_powerful_var > 24
 			}	
 		}
-	}
```

Comment:
So either this is wrong, or the indentation is wrong

Reply:
I forgot to remove that bracket when adding in the lines below, it was spitting out errors cus the current_value and on_monthly_pulse did not belong to any JE